site/en/reference/index.md

```diff
@@ -1,82 +1,30 @@
+Indexing, a process of oragnizing data, is a huge component of what makes it possible to efficiently query the million-, billion-, or even trillion-vector datasets that vector databases rely on. 
 
-Since accurate retrieval is usually very time-consuming, most of the vector index types of Milvus use ANNS (Approximate Nearest Neighbors Search). Compared with accurate retrieval, the core idea of ANNS is no longer limited to returning the most accurate result, but only searching for neighbors of the target. ANNS improves retrieval efficiency by sacrificing accuracy within an acceptable range.
+## **How does vector indexing work?**
 
-According to the implementation methods, the ANNS vector index can be divided into four categories:
+Similarity search engines work by comparing an input to a database to find objects that are most similar to the input. Indexing is the process of efficiently organizing data, and it plays a major role in making similarity search useful by dramatically accelerating time-consuming queries on large datasets. After a massive vector dataset is indexed, queries can be routed to clusters, or subsets of data, that are most likely to contain vectors similar to an input query. In practice, this means a certain degree of accuracy is sacrificed to speed up queries on really big vector data.
 
-- Tree-based index
-- Graph-based index
-- Hash-based index
-- Quantization-based index
-
-The following table classifies the indexes that Milvus supports:
-
-<table>
-<thead>
-  <tr>
-    <th>Supported index</th>
-    <th>Classification</th>
-    <th>Scenario</th>
-  </tr>
-</thead>
-<tbody>
-  <tr>
-    <td><a href="#FLAT">FLAT</a></td>
-    <td>N/A</td>
-    <td><ul>
-        <li>Has a relatively small dataset.</li>
-        <li>Requires a 100% recall rate. </li>
-        </ul></td>
-  </tr>
-  <tr>
-    <td><a href="#IVF_FLAT">IVF_FLAT</a></td>
-    <td>Quantization-based index</td>
-    <td><ul>
-        <li>High-speed query.</li>
-        <li>Requires a recall rate as high as possible.</li>
-        </ul></td>
-  </tr>
-  <tr>
-    <td><a href="#IVF_PQ">IVF_PQ</a></td>
-    <td>Quantization-based index</td>
-    <td></td>
-  </tr>
-  <tr>
-    <td><a href="#RNSG">RNSG</a></td>
-    <td>Graph-based index</td>
-    <td></td>
-  </tr>
-  <tr>
-    <td><a href="#HNSW">HNSW</a></td>
-    <td>Graph-based index</td>
-    <td></td>
-  </tr>
-  <tr>
-    <td><a href="#ANNOY">ANNOY</a></td>
-    <td>Tree-based index</td>
-    <td></td>
-  </tr>
-</tbody>
-</table>
 
 ## Vector field and index
 
-To improve query performance, you can specify an index type for each vector field. Currently, a vector field only supports one index type, Milvus will automatically delete the old index when switching the index type.
+To improve query performance, you can specify an [index type](index_selection.md) for each vector field. Currently, a vector field only supports one index type, Milvus will automatically delete the old index when switching the index type.
```

Comment:
one index type. Milvus automatically deletes the old index when switching the index type.